doc/rbd/rbd-config-ref.rst

```diff
@@ -146,45 +146,90 @@ RBD supports advanced features which can be specified via the command line when
 :Description: Layering enables you to use cloning.
 :Internal value: 1
 :CLI value: layering
+:Added in: 0.70
```

Comment:
Nit: some librbd versions state ```vX.Y.Z``` vs ```X.Y.Z``` -- perhaps standardize on the ```vX.Y.Z``` or just use the release name (i.e. ```Jewel```)